src/DataTable/FilterAndSortMenu.js

```diff
@@ -52,6 +53,17 @@ export default class FilterAndSortMenu extends React.Component {
     } else if (ccSelectedFilter === "notEmpty") {
       // manually set filter value (nothing is selected by user)
       filterValToUse = true;
+    } else if (ccSelectedFilter === "isEmpty") {
+      // manually set filter value (nothing is selected by user)
+      filterValToUse = false;
+    } else if (
+      ccSelectedFilter === "inList" ||
+      ccSelectedFilter === "notInList"
+    ) {
+      filterValToUse =
+        typeof filterValue === "string"
+          ? filterValue.split(".")
+          : filterValue.filter(op => op.value !== "").map(op => op.value);
```

Comment:
why are still supporting list filter value to be of type string? Don't we have the multiselect component now?